### DISCUSSION TITLE
Difference between ListView() and ListView.Builder()?

### DISCUSSION THREAD

**Discussion by flutterdevsb** [community/community#28063] [Mobile] [answered]
Difference between ListView() and ListView.Builder()?

**Comment by sthakrishnakumar** [comment, marked as answer]
Hey,
In general, ListView() renders all the items in its children at once.
Where as ListView.builder() renders the only items that are visible in the screen which means that this renders the item as per the requirement.
Here, Listview is efficient when we have less number of items where as Listview.builder is efficient for large number of items.

**Reply by flutterdevsb** [reply]
Thank you
